ptz-explainer.md

```diff
@@ -110,17 +113,22 @@ Having a separate PTZ permission allows the UA to differentiate between normal
 camera permissions and PTZ camera permissions as PTZ needs to be explicitly
 requested as an extension to the camera permission.
 
-UA may group both the camera permission request and the PTZ permission request.
-See mock below.
+The UA may group both the camera permission request and the PTZ permission
+request. See mock below.
```

Comment:
In a single permission prompt, for instance allowing user to pick and choose sub-permissions like PTZ

Reply:
That's basically what I'm saying below right?
> The UA may also provide a way for users to block PTZ directly from the prompt.